test/statewide_test_repository_test.rb

```diff
@@ -8,5 +8,72 @@ def test_starts_off_with_empty_statewide_tests
   end
 
   def test_can_load_data
+    str = StatewideTestRepository.new
+    str.load_data({
+      statewide_testing: {
+        third_grade: "./test/sample_third_grade.csv"
+      }
+    })
+    statewide_test = str.statewide_test_exists("Colorado")
+    assert_equal "COLORADO", statewide_test.name
+  end
+end
+
+class StatewideTestRepositoryIntegrationTest < Minitest::Test
+  def test_can_load_data_into_enrollment_repository
+    str = StatewideTestRepository.new
+    str.load_data({
+      :statewide_testing => {
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.